.php_cs

```diff
@@ -0,0 +1,34 @@
+<?php
+$header = <<<'EOF'
+
```

Comment:
This blank line needs to be removed, and the spaces before the other header lines need to be removed, as well. It should be:

```php
$header = <<<'EOF'
This file is part of the `liip/LiipImagineBundle` project.

(c) https://github.com/liip/LiipImagineBundle/graphs/contributors

For the full copyright and license information, please view the LICENSE.md
file that was distributed with this source code.
EOF;
```